<?xml version="1.0"?>
<launch>
    <!-- Nodelet manager -->
    <node pkg="nodelet" type="nodelet" name="test_nodelet_manager" args="manager" cwd="node" output="screen"/>

    <!-- Camera restamper -->
    <node pkg="nodelet" type="nodelet" name="camera_restamp_nodelet"
      args="load ig_handle/camera_restamp_nodelet test_nodelet_manager" >
      <param name="max_buffer_size" value="100"/>
      <param name="data_topic" value="/F1/image_raw"/>
      <param name="time_topic" value="/F1/cam_time"/>
    </node>

    <!-- IMU restamper -->
 <node pkg="nodelet" type="nodelet" name="imu_restamp_nodelet"
        args="load ig_handle/imu_restamp_nodelet test_nodelet_manager" >
    <param name="max_buffer_size" value="2000"/>
    <param name="data_topic" value="/imu/data"/>
    <param name="time_topic" value="/imu/imu_time"/>
  </node>

</launch>